core-web/src/main/resources/META-INF/viewGenerator.tld

```diff
@@ -1087,34 +1087,13 @@
       <rtexprvalue>false</rtexprvalue>
       <type>boolean</type>
     </attribute>
-    <attribute>
-      <description>URL to be used to export data.</description>
-      <name>exportURL</name>
-      <required>false</required>
-      <rtexprvalue>false</rtexprvalue>
-      <type>java.lang.String</type>
-    </attribute>
-    <attribute>
-      <description>Defines if lines can be explicitly sorted</description>
-      <name>sortableLines</name>
-      <required>false</required>
-      <rtexprvalue>true</rtexprvalue>
-      <type>boolean</type>
-    </attribute>
     <attribute>
       <description>Defines number of lines displayed per page</description>
       <name>numberLinesPerPage</name>
       <required>false</required>
       <rtexprvalue>true</rtexprvalue>
       <type>java.lang.Integer</type>
     </attribute>
-    <attribute>
-      <description>Defines the javascript callback to call on a line move action</description>
-      <name>moveLineJsCallback</name>
```

Comment:
This is yet used (favorite links for example)

Reply:
Yes. Only in two jsp/tag. Shit